_drafts/oryx/orchestration/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Application orchestration
+description: Orchestration of the Oryx Application
+template: concept-topic-template
+---
+
+# Orchestrate an Oryx Application
+
+Any Oryx application starts with the application orchestration.
+It allows you to bootstrap and configure your application from reusable bits and pieces such as:
+
+- [Features](./app-feature.md)
+<!-- TODO: Link to components -->
+- Components
+<!-- TODO: Link to providers -->
+- Providers
+<!-- TODO: Link to themes -->
+- Themes
+- [Resources](./resources.md)
+- [Environment](./app-environment.md)
+
+Application orchestration is used to configure and customize your Oryx application to your particular needs.
+As Oryx is a framework it provides many different pieces of functionality for different use cases such as B2B, B2C, Backoffice, Fulfillment, etc. And to be able to select specific pieces from Oryx you can use orchestration to pick the features that you need and also customize them as required.
+
+Application orchestration is managing what pieces of functionality will be available in the Oryx application and how they are going to be loaded. For example components will be lazy-loaded when they are used on the page but services will be eagerly loaded during application startup.
+
+To start using orchestration you need to import [`appBuilder`](./app-builder.md) from the `@spryker-oryx/core` and then you can start composing your own set of features with it by adding features, theme, etc. to your application.
+
+`appBuilder` uses a chain pattern where each customization is added via respective `.with*` method.
+This is the minimum boilerplate code that your application requires in order to work.
+
+Once you start building more complex use-cases in your Oryx application it is recommended to create your own feature set instead of using a preset or extend existing feature set.
+
+Application builder allows you to compose and customise different pieces of functionality in your Oryx application.
+
+It is a chainable and pluggable builder which supports the following built-ins:
+
+- [Features](./app-feature.md) (`withFeature`)
+<!-- TODO: Link to components -->
+- Components (`withComponents`)
+<!-- TODO: Link to providers -->
+- Providers (`withProviders`)
+<!-- TODO: Link to themes -->
+- Themes (`withTheme`)
+- [Options](#options) (`withAppOptions`)
+- [FeatureOptions](./app-feature.md) (`withOptions`)
+- [Environment](./app-environment.md) (`withEnvironment`)
+- [Resources](./resources.md) (`withResources`)
+- [Plugins](./app-plugins.md) (`with`)
+
+All of the above are built-in plugins but you can also add your custom plugins by using `with` API:
```

Comment:
In the intro sentence, you say these are `built-ins`. Here, they are called `built-in plugins`. So, what are they?

Reply:
Does the colon in the end of the sentence refer to the example in the next secion?